docker/gateway/tls/docker-compose.yaml

```diff
@@ -0,0 +1,212 @@
+# Best used if called by 'tls-download-mutinynet.sh' script
+# fedimint.my-super-host.com should be replaced by a domain controlled by you and all DNS below should redirect to the IP of your server.
+# Domains:
+# - fedimint.my-super-host.com
+# - fedimintd.fedimint.my-super-host.com
+# - guardian-ui.fedimint.my-super-host.com
```

Comment:
these can be removed from the gateway scripts, might want to check all of the yaml comments

Reply:
removed in next commit